arangograph/backups.md

```diff
@@ -38,20 +46,58 @@ It's also possible to create a backup on demand. To do this, click **Back up now
 
 ![Back up Now Dialog](images/arangograph-back-up-now-dialog.png)
 
+If you want to manually copy a backup to a different region than the default
+one, first ensure that the **Upload backup to storage** option is enabled.
+Then, highlight the backup row and use the
+**Copy backup to a different region** button from the **Actions** column. 
+Once the copy of the default backup is created in the region of your choice,
+you can copy it to a different region as well. The source backup ID from
+which the copy is created is displayed in the **Copied from Backup** column.
+
+![Copy backup to a different region](images/arangograph-copy-backup-different-region.png)
+
+![Multiple Backups](images/arangograph-multiple-backups.png)
+
 ### Uploading backups
 
-By default a backup is not uploaded to the cloud, instead it remains on the
+By default, a backup is not uploaded to the cloud, instead it remains on the
 servers of the deployment. To make a backup that is resilient against server
-(disk) failures, upload the backup to cloud storage. Uploaded backups are
+(disk) failures, upload the backup to cloud storage. 
+
+When the **Upload backup to cloud storage option** is enabled, the backup is
+preserved for a long time and does not occupy any disk space on the servers.
+This also allows copying the backup to different regions and it can be
+configured in the **Multiple region backup** section.
+
+Uploaded backups are
 required for [cloning](#how-to-clone-deployments-using-backups).
 
+## Multi-region backups
+
+{% hint 'info' %}
+The multi-region backup feature is only available for the ArangoGraph
+Enterprise tier. To upgrade to the Enterprise tier,
+[get in touch](https://www.arangodb.com/contact/){:target="_blank"}
+with the ArangoDB team.
+{% endhint %}
+
+Using the multi-region backup feature, you can store backups in multiple regions
+simultaneously either manually or automatically as part of a **Backup policy**.
+If a backup created in one region goes down, it is still available in other
+regions, significantly improving reliability. 
+
+Multiple region backup is only available when the
+**Upload backup to cloud storage** option is enabled.
+
+![Multiple Region Backup](images/arangograph-multi-region-backup.png)
+
 ## How to restore backups
 
 To restore a database from a backup, highlight the desired backup and click the restore icon.
 
 {% hint 'warning' %}
 All current data will be lost when restoring.
```

Comment:
What is meant by this exactly? (All current data will be lost). If some data loss may happen, we also need to say this in the UI warning - this currently only says the database will be unavailable for some time. Please check and adjust the UI message/docs as needed.

Reply:
@nikhil-varma maybe you can answer this question?

Reply:
Assume you create a backup at t1, then at t2 (after already inserting new data) you are restore the backup from t1, the data inserted after that time will be 'lost'. if you want to go back to that data you can create a backup just before you restore a backup, that way, you can go back to t2 again.